scripts/global/tune_player.js

```diff
@@ -169,9 +171,7 @@ var createTunePlayer = function(audioContext, bpm) {
 
   var getStepDuration = function(instrument, bpm) {
     if(stepDuration) return stepDuration;
-	stepDuration = (instrument == 'sampler') ? 1 / (bpm / 90) : 1 / (bpm / 60);
-	//alert(instrument.toString());
-    //stepDuration = 1 / (bpm / 90);	//This affects the tune's speed in both the editor and hourly chime. Find a better way to slow down the bells.
+	stepDuration = (instrument.instrumentName == 'sampler') ? 1 / (bpm / 90) : 1 / (bpm / 60);
```

Comment:
Strangely, this check never returns true, as the string variable "instrumentName" apparently doesn't exist when it checks it, returning "undefined" for both its contents and _typeof_. The sampler object is always created before this could be checked, so I have no clue why it isn't returning "sampler" or "booper". Does anyone know what's going wrong here?